### PR TITLE
Add cdn field to dev page

### PR DESCRIPTION
**Pull request by danieljackins**
Adds a field to change the CDN the test app loads from
image

### CODE REVIEW

**Comment by silesky** [comment]
I actually have some relevant changes in my PR! https://github.com/segmentio/analytics-next/pull/424/files#diff-45418fb36b97a6285f0f01af612da254af14469374e3e255a735e1a7d972049f

**Review by chrisradek — APPROVED** [review]
Approved! Next will be generating the fields for __ALL__ of the possible settings ;)

File: example/pages/index.tsx [hunk 1]
```diff
@@ -124,10 +143,18 @@ export default function Home(): React.ReactElement {
       <form
         onSubmit={(e) => {
           e.preventDefault()
-          setSettings({ writeKey: writeKey })
-          fetchAnalytics()
+          setSettings({ cdnURL: url, writeKey: writeKey })
```
Comment (silesky) [inline example/pages/index.tsx:146]: much better!

File: example/pages/index.tsx [hunk 2]
```diff
@@ -31,13 +36,27 @@ const jsontheme = {
   base0F: '#a3685a',
 }
```
Comment (silesky) [inline example/pages/index.tsx:38]:
```suggestion

// This function runs on dependency change but not on initial render.
```

**Pull request merged**